### ISSUE TITLE
The graph is opened for some random submodule, not the one I clicked

### ISSUE THREAD

**Issue by adalisk-emikhaylov** [phil294/git-log--graph#42]
I'm working with multiple submodules.

When I click any of those buttons:

![image](https://github.com/phil294/git-log--graph/assets/69473580/c91253a6-105f-42e0-9fe0-8ba5f48cf130)

The graph is always opened for some fixed submodule, which isn't even the main repo.

Instead, the button should open the graph for the actual submodule I clicked.

**Comment by phil294** [comment]
so apparently, the repo indices are not reliable. You had probably rearranged them at some point. I changed it to path comparison, so it should work now.

**Comment by adalisk-emikhaylov** [comment]
Interestingly, I have another local copy of this repository, and there git-log--graph always opens the graph for the main repository (even if I press the 'graph' button on a submodule).

**Comment by phil294** [comment]
I *hope* this is fixed now. I couldn't really reproduce it, so it was guesswork. Please give it a try!

**Comment by adalisk-emikhaylov** [comment]
@phil294 Nope, still doesn't work. Tried it on three copies of the same repo at different paths, and while two opened the main repo, the third one still opened a random submodule.

Also, the UI has multiple buttons to open the graph (see picture in the first post), one per submodule, and all of the still open the main repository, not the respective submodules.

**Comment by lens0021** [comment]
In v0.1.11, I am also experiencing this even though I have opened four repositories, without any submodule.

**Comment by phil294** [comment]
@lens0021 latest version should be 0.1.12 though (few minutes ago). Perhaps a refresh will help?

**Comment by lens0021** [comment]
I've tried the latest version. Here is my reproduction steps:

```bash
for num in 1 2 3; do
 mkdir "repe$num"
 cd "repe$num"
 git init
 touch empty
 git add .
 git commit -m "commit$num"
 cd ..
done
code repo*
```

And result:

[Screencast from 2023-08-01 01-37-34.webm](https://github.com/phil294/git-log--graph/assets/28209361/0021ad34-c163-4b90-bbd7-518f147ecbf5)

**Comment by phil294** [comment]
hi @lens0021, thanks for the precise reproduction steps. Interestingly my fix only worked with [VSCodium](https://github.com/VSCodium/vscodium) builds because the build process is different and certain properties aren't uglified. Annoying AF... but it should be fixed now with v0.1.14 @adalisk-emikhaylov
